### PR TITLE
feat(chat): change playback start screen(issue #600, #834)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
Issues:

- #600 
- #834 

image

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by denys-kolomiitsev** [comment]
/deploy-review

>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/ConversationSettingsModels.tsx [hunk 1]
```diff
@@ -64,14 +63,29 @@ export const ConversationSettingsModel = ({
     [dispatch, onModelSelect],
   );
 
+  const playbackModelID =
+    conversation.playback?.messagesStack[
+      conversation.playback.activePlaybackIndex
+    ]?.model?.id ?? conversation.model.id;
+
+  const playbackModelName = modelsMap[playbackModelID]?.name;
```
Comment (mikitabut) [inline apps/chat/src/components/Chat/ConversationSettingsModels.tsx]: `|| playbackModelID`

**Review by mikitabut — DISMISSED** [review]

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/ConversationSettingsModels.tsx (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/components/Chat/ConversationSettingsModels.tsx]: Changed as suggested.

**Review by mikitabut — DISMISSED** [review]

File: apps/chat/src/components/Chat/ModelList.tsx [hunk 2]
```diff
@@ -57,13 +60,17 @@ const ModelGroup = ({
   return (
     <div
       className={classNames(
-        'flex cursor-pointer items-center gap-3 rounded border px-3 py-2 hover:border-hover',
-        selectedModelId === currentEntity.id
+        'flex items-center gap-3 rounded border px-3 py-2 hover:border-hover',
+        !disabled && selectedModelId === currentEntity.id
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ModelList.tsx:64]: I suppose we still need to show model as selected

File: apps/chat/src/components/Chat/ConversationSettingsModels.tsx [hunk 3]
```diff
@@ -98,12 +112,14 @@ export const ConversationSettingsModel = ({
             displayCountLimit={
               unavailableModelId ? RECENT_MODELS_COUNT - 1 : RECENT_MODELS_COUNT
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ConversationSettingsModels.tsx]: you show fake model and need to decrease displayCountLimit
```suggestion
              isPlayback || unavailableModelId ? RECENT_MODELS_COUNT - 1 : RECENT_MODELS_COUNT
```

File: apps/chat/src/components/Chat/ConversationSettingsModels.tsx [hunk 4]
```diff
@@ -64,14 +63,29 @@ export const ConversationSettingsModel = ({
...
+
   return (
     <div className="w-full" data-qa="entity-selector">
       <div className="mb-4">{t('Talk to')}</div>
 
       <div className="flex flex-col gap-3" data-qa="recent">
         <div className="grid grid-cols-1 gap-3">
-          {replay.isReplay && (
-            <ReplayAsIsButton replay={replay} conversationId={conversationId} />
+          {conversation.playback?.isPlayback && (
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/ConversationSettingsModels.tsx]: use `isPlayback` here
```suggestion
          {isPlayback && (
```

File: apps/chat/src/components/Chat/ConversationSettingsModels.tsx (hunk 3 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/components/Chat/ConversationSettingsModels.tsx]: Thank you for the suggestion!
Changed.

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-826.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**